### PR TITLE
Slightly simplify autotest

### PR DESCRIPTION
**Pull request by okurz**

### CODE REVIEW

**Review by Martchus — COMMENTED** [review]
Looks good, I have only one suggestion.

File: autotest.pm [hunk 1]
```diff
@@ -338,9 +336,7 @@ sub start_process () {
 sub query_isotovideo ($cmd, $args = undef) {
     # deep copy
     my %json;
-    if ($args) {
-        %json = %$args;
-    }
+    %json = %$args if $args;
```
[the remark this answers is not in the document]
Comment (perlpunk) [reply autotest.pm:339]: or
```suggestion
    my %json = %{$args || {}};
```

**Review by Martchus — APPROVED** [review]

File: autotest.pm (hunk 1 above)
[the remark this answers is not in the document]
Comment (okurz) [reply autotest.pm]: Need to fix this up manually

**Review by Wabri — COMMENTED** [review]
Code coverage decreases

**Comment by okurz** [comment]
> Code coverage decreases

I don't see that. I only see changes in so far uncovered lines which was expected and still to be done by you ;)

**Comment by Wabri** [comment]
LGTM!

So is it ok to merge uncovered code when there is a ticket that should handle the coverage?

**Comment by okurz** [comment]
> LGTM!
> 
> So is it ok to merge uncovered code when there is a ticket that should handle the coverage?

in exceptions yes. But here my change should actually help by making the lines easier to cover.

**Pull request merged**